### PR TITLE
#1307 Types in runtime

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Ref: #1307

---

### Focus of this PR:
This PR focuses on adding the `type()` method to several classes in the `eo-runtime` package.

### Detailed summary:
- Added the `type()` method to the following classes: `PhSafe`, `PhConst`, `PhNamed`, `PhLocated`, `PhImmovable`, `PhOnce`, `Data`, `PhLogged`, `PhPackage`, `PhWith`, `PhMethod`, and `PhDefault`.
- The `type()` method returns the type of the object as a `String`.
- The `type()` method is implemented by calling the `type()` method of the `origin` object in each class.
- Added corresponding unit tests for the `type()` method in the `PhCopyTest`, `PhPackageTest`, `PhWithTest`, `PhMethodTest`, and `PhDefaultTest` classes.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by maxonfjvipon** [comment]
@Graur please review this one

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/test/java/org/eolang/PhDefaultTest.java [hunk 1]
```diff
@@ -228,6 +229,53 @@ void readsMultipleTimesThroughAttribute() {
         );
     }
 
+    @Test
+    void hasTheSameTypeWithBoundedData() {
+        Phi five = new EOint(Phi.Φ);
```
[the remark this answers is not in the document]
Comment (maxonfjvipon) [reply eo-runtime/src/test/java/org/eolang/PhDefaultTest.java:234]: @Graur sure

**Comment by maxonfjvipon** [comment]
@Graur please have a look one more time

**Review by Graur — APPROVED** [review]
@maxonfjvipon Good one
@yegor256 Please have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**